core/com.dexels.navajo.core/src/com/dexels/navajo/server/Dispatcher.java

```diff
@@ -350,13 +350,6 @@ private final Navajo dispatch(Access access) throws Exception {
         try {
             ServiceHandler sh = handlerFactory.createHandler(navajoConfig, access, simulationMode);
 
-            // If recompile is needed ALWAYS set expirationInterval to -1.
-            // ALSO I DO NOT WANT CACHECONTROLLER DEPENDENCY @ THIS POINT.
-            long expirationInterval = CacheController.getInstance().getExpirationInterval(access.rpcName);
-            if (expirationInterval > 0 && sh.needsRecompile( access )) {
-                expirationInterval = -1;
-            }
-
             // Remove password from in to create password independent
             // persistenceKey.
             in.getHeader().setRPCPassword("");
```

Comment:
@aschoneveld  Is this line still needed; the comment above and the rest of your changes suggest it is no longer needed; or am I missing something?

Reply:
Yes, that line can be removed. I will do that.